go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/fjl/gencodec v0.0.0-20220412091415-8bb9e558978c
 	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5
 	github.com/gballet/go-libpcsclite v0.0.0-20190607065134-2772fd86a8ff
-	github.com/gballet/go-verkle v0.0.0-20230414192453-2838510d5ee0
+	github.com/gballet/go-verkle v0.0.0-20230420192103-fe912821ad59
```

Comment:
Pointing to last commit of https://github.com/gballet/go-verkle/pull/343

Reply:
is there any reason that PR is still in draft, btw?

Reply:
Mostly to have some extra information after seeing how would fit "this side" of the story.
Opened for review; not sure if you want to have an extra look, but from my side it's fine to review & merge.

After that, I can update `go.mod` to the latest master.